docs/installation/octopus-server-linux-container/docker-compose-linux.md

```diff
@@ -127,6 +127,23 @@ docker-compose --project-name Octopus up -d
 
 When both containers are healthy, you can browse directly to `http://localhost:8080` from your host machine.
 
+## Trusting custom/internal Certificate Authority (CA)
+Octopus Server can interface with a number of external sources (feeds, git repos, etc...), those sources are often configured to use SSL/TLS for secure communication.  It is common for organizations to have their own CA servers for their internal networks.  A CA server can issue SSL certificates that can be used on internal resources, such as build servers or internally hosted applications without the need to purchase from a third-party vendor.  Technologies such as Group Policy Objects (GPO) can configure machines (servers and clients) to automatically trust the CA so users don't have trust them manually, however, this is not inherited to containers.  When attempting to configure a connection to an external resource with an untrusted CA, you'll most likley encounter
+
+```
+Could not connect to the package feed The SSL connection could not be established, see inner exception. The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot
+```
```

Comment:
```suggestion
```text
Could not connect to the package feed. The SSL connection could not be established, see inner exception. The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot
```
```